AbstractDiffractometer and AbstartSampleView

beteva opened the pull request:
This PR replaces #1881.

Changes due to AbstractDiffractometer and AbstartSampleView new implementation. Dependant on mxcube/mxcubecore/pull/1479

Commit:
Change of head and chip definition

Commit:
Changes due to new AbstractDiffractometer/AbstractSampleView HOs

Commit:
new configuration files

Commit:
Change phi to omega

axelboc commented:
<img width="164" height="74" alt="image" src="https://github.com/user-attachments/assets/3e80e3cc-566f-42fa-9b4b-170ff645b1c8" />

:tada: :partying_face:

Commit:
Make sample_view_adapter use sample_view instaed of diffractometer.
Use reject_centring instead of cancel_centring when abort the centring.
Change phi to omega in the GridForm

Commit:
Replace phi by omega.

Commit:
Make _centring_add_current_point more robust.
Change logic in _centring_started.
Add missing float type for sample changer input model.

Commit:
in_plate_mode is property

beteva committed:
Update pyproject.toml

Use the 2.1.0 mxcubecore version

marcus-oscarsson committed:
Poetry lock

Commit:
Change test to match corresponding demo setup.

Commit:
Phases have changed labels.

Commit:
Omega direction changed to be compliant with the e2e tests.

Commit:
tests run better on chrome.

Commit:
Change yaml files to reflect the new diffractometer and sample_view.

Commit:
Restore firefox for testing.

Commit:
Bug fix - unnecessary check removed.

Commit:
Re-format!!!!!!!!!!!!!!!!

beteva committed:
Update pyproject.toml

Commit:
update poetry lock file as required by the CI
